src/Common/src/Interop/Unix/libcurl/Interop.libcurl_multi.cs

```diff
@@ -52,5 +53,8 @@ internal static partial class libcurl
         [DllImport(Interop.Libraries.LibCurl)]
         public static extern IntPtr curl_multi_strerror(
             int code);
+
+        [DllImport(Libraries.CryptoNative)]
```

Comment:
Since this is not calling into libcurl, it should not be in a libcurl Interop file.  Move to System.Security.Native.Cryptography\